DuckDuckGo/NetworkProtectionStatusViewModel.swift

```diff
@@ -100,6 +101,30 @@ final class NetworkProtectionStatusViewModel: ObservableObject {
     private let errorObserver: ConnectionErrorObserver
     private var cancellables: Set<AnyCancellable> = []
 
+    // MARK: - Tips
+
+    @Published
+    var showAddWidgetEducationView: Bool = false
+
+    @Published
+    var vpnEnabledTips: TipGrouping = {
+        if #available(iOS 18.0, *) {
+            return TipGroup(.ordered) {
+                VPNChangeLocationTip()
+                VPNUseSnoozeTip()
+                VPNAddWidgetTip()
+            }
+        } else if #available(iOS 17.0, *) {
+            return LegacyTipGroup(.ordered) {
+                VPNChangeLocationTip()
+                VPNUseSnoozeTip()
+                VPNAddWidgetTip()
+            }
+        } else {
+            return EmptyTipGroup()
+        }
```

Comment:
This is great because we'll be using the default thing in iOS 18+, my backport in iOS 17 and a glorified no-op in lower OS versions.